test/test_ember_handlebars_template.rb

```diff
@@ -145,6 +145,26 @@ def test_should_have_module_name_with_AMD_output
     end
   end
 
+  def test_compile_raw_template_and_strip_the_raw_suffix
+    with_ember_template 'Handlebars' do
+      asset = @env['app/templates/hi.raw.js']
```

Comment:
I think this file should be accessible without `.raw` extension.

Could you update [`Ember::Handlebars::Template.setup`](https://github.com/tricknotes/ember-handlebars-template/blob/v0.6.0/lib/ember/handlebars/template.rb#L23-L25) with `.raw` extension processor?
Such as the following:
``` ruby
env.register_engine '.raw.hbs', self, mime_type: 'application/javascript'
```